Libplanet.Store/TrieStateStore.Commit.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography;
+using Bencodex;
+using Bencodex.Types;
+using Libplanet.Common;
+using Libplanet.Store.Trie;
+using Libplanet.Store.Trie.Nodes;
+
+namespace Libplanet.Store
+{
+    public partial class TrieStateStore
+    {
+        private static readonly Codec _codec = new Codec();
+
+        /// <inheritdoc cref="IStateStore.Commit"/>
+        public ITrie Commit(ITrie trie)
```

Comment:
IMO we need some tests for this method.

Reply:
It's mostly handled by old `ITrie.Commit()`. I suppose we need to move them. 😕

Reply:
Just added some tests.